examples/coco/.tensorleap/dataset.py

```diff
@@ -224,7 +221,7 @@ def metadata_brightness(idx, data):
 dataset_binder.set_input(input_image, 'images', DatasetInputType.Image, 'image')
 
 dataset_binder.set_ground_truth(ground_truth_mask, 'images', ground_truth_type=DatasetOutputType.Mask, name='mask',
-                                labels=['background', 'person', 'bicycle', 'car'], masked_input="image")
+                                labels=categories, masked_input="image")
```

Comment:
We actually have 3 categories (and not 2):
```suggestion
                                labels=['background', 'person', 'car'], masked_input="image")
```